src/main/java/fr/techad/edc/popover/internal/swing/components/IconButtonListener.java

```diff
@@ -112,6 +112,7 @@ private void openPopover(int x, int y) {
                 JComponent jBodyComponent = contextualContentComponentBuilder.setContextItem(contextItem).setBackgroundColor(helpConfiguration.getBackgroundColor()).enableArticle(helpConfiguration.isShowArticle()).build();
                 JComponent jTitleComponent = contextualTitleComponentBuilder.setContextItem(contextItem).setBackgroundColor(helpConfiguration.getBackgroundColor()).enableTitle(helpConfiguration.isShowTitle()).build();
                 JComponent jBodyComponent = contextualContentComponentBuilder.setContextItem(contextItem).setBackgroundColor(helpConfiguration.getBackgroundColor()).build();
+                JComponent jTitleComponent = contextualTitleComponentBuilder.setContextItem(contextItem).setBackgroundColor(helpConfiguration.getBackgroundColor()).setShowTitle(helpConfiguration.isShowTitle()).build();
```

Comment:
Check this class but this file wil not compile. You defined several JComponent with the same name. Avoid to merge with github editor ;)